spring-boot-samples/spring-boot-sample-web-secure/src/main/java/sample/web/secure/SampleWebSecureApplication.java

```diff
@@ -60,22 +58,36 @@ public static void main(String[] args) throws Exception {
 		new SpringApplicationBuilder(SampleWebSecureApplication.class).run(args);
 	}
 
-	@Configuration
-	@Order(SecurityProperties.ACCESS_OVERRIDE_ORDER)
-	protected static class ApplicationSecurity extends WebSecurityConfigurerAdapter {
+	@EnableWebSecurity
```

Comment:
I don't think this should be using `@EnableWebSecurity`. That's the purpose of `spring-boot-sample-web-secure-custom`.